scripts/git-org-name.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+die() {
+  local _return="${2:-1}"
+  echo "$1" >&2
+  exit "${_return}"
+}
+
+# Returns the git orgname from a given git url (both https and git)
+# Example: git@github.com:openebs/mayastor.git => openebs
+urlOrgName() {
+  local gitUrl="$1"
+  if echo $gitUrl | grep -qa "^git@"; then
+    echo "$gitUrl" | awk -F/ '/git/ { print $1 }' | cut -d':' -f2
+  elif echo $gitUrl | grep -qa "^https://"; then
+    echo "$gitUrl" | awk -F/ '/https/ { print $4 }'
+  else
+    die "Unknown git url type: '$gitUrl'"
+  fi
+}
+
+orgName() {
+  local originUrl=$(git config --get remote.origin.url)
```

Comment:
Is the remote always guaranteed to be 'origin'? Is this meant for automation only? I call the openebs org 'upstream' in my dev setup.

Reply:
Since we clone , I would say yes.

Reply:
It's meant for the CI so we can name dockerhub credentials and also dockerhub repository name.
We can add a manual repository override for local image building if required.